orca-core/src/main/groovy/com/netflix/spinnaker/orca/listeners/ExecutionPropagationListener.groovy

```diff
@@ -26,20 +26,8 @@ import static com.netflix.spinnaker.orca.ExecutionStatus.*
 @Slf4j
 @CompileStatic
 class ExecutionPropagationListener implements ExecutionListener {
-  private boolean isBeforeJobEnabled = false
-  private boolean isAfterJobEnabled = false
```

Comment:
I like that we're getting rid of this stuff but does it have anything directly to do with this change?